src/constants/function_runtime.ts

```diff
@@ -16,7 +17,10 @@ export class FunctionRuntimeUtil {
             return FunctionRuntimeConstant.None;
         }
 
-        const key: string = language.charAt(0).toUpperCase() + language.toLowerCase().slice(1);
+        let key: string;
```

Comment:
Do you mind create a new test file in <project_root>/tests/utils/function_runtime.spec.ts?
In the test cases, please test the following three scenarios:
1. dotnet --expects--> Dotnet
2. dotnet-isolated --expects--> DotnetIsolated
3. invalid --expects--> None

Reply:
I've added the tests, but the 3rd scenario expects to throw with the current implementation.